dashboard/context_processors.py

```diff
@@ -4,14 +4,15 @@
 import logging
 logger = logging.getLogger(__name__)
 
+from django.http import HttpRequest, JsonResponse
 
-def get_git_version_info(request):
+def get_git_version_info(request: HttpRequest) -> JsonResponse:
     return {'git_version': utils.get_git_version_info()}
 
 
-def get_myla_globals(request):
+def get_myla_globals(request: HttpRequest) -> JsonResponse:
     return {'myla_globals': utils.get_myla_globals(request.user)}
 
 
-def last_updated(request):
-    return {'last_updated': db_util.get_canvas_data_date()}
+def last_updated(request: HttpRequest) -> JsonResponse:
+    return {'last_updated': db_util.get_canvas_data_date()}
```

Comment:
The value returned by these seems to be `Dict`, though it's possible Django does something extra with context processors. The specific type of the requests is `<class 'django.core.handlers.wsgi.WSGIRequest'>`. It's hard to say exactly what to put for these, since these specific object types won't always be that clear to users.

Reply:
It looks like the https://pypi.org/project/django-hint/ package has support for this, I could include that and change it.